core-library/src/main/java/org/silverpeas/core/contribution/content/form/displayers/WysiwygFCKFieldDisplayer.java

```diff
@@ -508,10 +517,9 @@ public void index(FullIndexEntry indexEntry, String key, String fieldName, TextF
     }
   }
 
-  public String duplicateContent(Field field, FieldTemplate template, ForeignPK from, ForeignPK to,
+  public String duplicateContent(FieldTemplate template, ForeignPK from, ForeignPK to,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Remove this use of "ForeignPK"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Remove this use of "ForeignPK"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)